srcpkgs/exfat-utils/template

```diff
@@ -1,11 +1,12 @@
 # Template file for 'exfat-utils'
 pkgname=exfat-utils
 version=1.3.0
-revision=1
+revision=2
 build_style=gnu-configure
 short_desc="Utilities for exFAT file system"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/relan/exfat"
 distfiles="${homepage}/releases/download/v${version}/${pkgname}-${version}.tar.gz"
 checksum=dfebd07a7b907e2d603d3a9626e6440bd43ec6c4e8c07ccfc57ce9502b724835
+conflicts="exfatprogs"
```

Comment:
You don't need to add a conflict to two packages, this just results in two error messages instead of one.